CONTRIBUTING.md

```diff
@@ -1,4 +1,32 @@
-# Contributing
+# Contributing to Telegraf
+
+Telegraf is a large open-source project and there are multiple ways to contribute.
+There are many ways to get involved in the Telegraf project! From opening issues, creating pull requests, to joining the conversation in Slack. We would to see you contribute your expertise and join our community. To get started review this document to learn best practices.
+
+![tiger](assets/GopherAndTiger.png "tiger")
+
+## Opening Issues
+
+### Bug reports
+
+Before you file an issue, please search existing issues in case it has already been filed, or perhaps even fixed. If you file an issue, please include the following.
```

Comment:
```suggestion
Before you file an issue, please search existing issues in case it has already been filed, or perhaps even fixed. If you file an issue, please ensure you include all the requested details (e.g. Telegraf config and logs, platform, etc.)
```

No need to list out all the items, we have a bug form for them to fill out and we don't want to keep both places in sync.